modules/renter/contractor/formcontract.go

```diff
@@ -363,6 +367,7 @@ func (c *Contractor) formContracts(a modules.Allowance) error {
 			// negotiate with because they'll probably be more expensive than
 			// we can afford.
 			c.log.Println("WARN: failed to negotiate contract:", h.NetAddress, err)
```

Comment:
On the contrary, `formContract` should be called only when setting the allowance, and maybe when renewing.
The reason the renter log gets so big is because it will try to repair something, fails, logs the error, and then try again 5 seconds later. That's definitely bad.